CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Gravity is an open source project.
+
+It is the work of tens of contributors. We appricate your help!
+
+We follow a [code of conduct](./CODE_OF_CONDUCT.md).
+
+
+## Filing an Issue
+
+Security issues should be reported directly to security@gravitational.com.
```

Comment:
Do we have a public security page/policy we should point folks towards, or is security@gravitational.com the right endpoint?

Reply:
security@gravitational.com is currently the correct contact endpoint. We also have https://gravitational.zendesk.com/hc/en-us/articles/115015950248-Security-Disclosure with some bare bones information.

I do think we should put a dedicated page on the main site, but haven't really pushed for it or drafted any content.